.github/workflows/ci.yml

```
@@ -48,32 +48,49 @@ jobs:
     runs-on: ubuntu-24.04 # latest
     strategy:
       matrix:
-        compiler:
-          - clang-3
-          - clang-6
-          - clang-8
-          - clang-9
-          - clang-10
-          - clang-11
-          - clang-15
-          - clang-17
-          - gcc-4.8
-          - gcc-5
-          - gcc-6
-          - gcc-7
-          - gcc-8
-          - gcc-11
-          # - gcc-13 TODO: figure out why its not passing
+        variant:
```

trivial: old variable name was useful
```suggestion
        compiler:
```